py/circuitpy_mpconfig.h

```diff
@@ -463,6 +463,13 @@ extern const struct _mp_obj_module_t ustack_module;
 #define USTACK_MODULE
 #endif
 
+#if CIRCUITPY_PEW
```

Comment:
Are there any dependencies between `CIRCUITPY_PEW`, `CIRCUITPY_GAMEPAD`, and `CIRCUITPY_STAGE`? If so, check validity here with some `if`'s and `#error`'s. I wasn't sure when I did this refactoring, so I didn't do such checks, but you know better.

Reply:
There is no dependency. ``CIRCUITPYTHON_PEW`` is completely independent from the others.